### ISSUE TITLE
Detecting endianness doesn't work on Aarch64

### ISSUE THREAD

**Issue by xfix** [mdavidsaver/setuptools_dso#30]
I'm trying to package setuptools_dso for NixOS (https://github.com/NixOS/nixpkgs/pull/269695), and endianness detection fails on Aarch64.

```
======================================================================
FAIL: test_info (setuptools_dso.test.test_probe.TryCompile.test_info)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/src/setuptools_dso/test/test_probe.py", line 76, in test_info
    self.assertIn(info.endian, ("little", "big"))
AssertionError: None not found in ('little', 'big')
----------------------------------------------------------------------
Ran 6 tests in 3.320s
```

On https://godbolt.org/, it can be seen that `__ARMEL__` variable doesn't exist when compiling for ARM64.

**Comment by mdavidsaver** [comment]
From some [investigation](https://sourceforge.net/p/predef/wiki/Endianness/), I think an answer is testing for `__AARCH64EB__` or `__AARCH64EL__`.

**Comment by mdavidsaver** [comment]
@xfix Please re-test with 45ae5d68275964688cff61b8bae9e50be9d410bf.

**Comment by xfix** [comment]
This patch appears to work on https://github.com/NixOS/nixpkgs/pull/269695 :).

**Comment by mdavidsaver** [comment]
Prompt feedback is appreciated.  Released as [2.10](https://pypi.org/project/setuptools-dso/2.10/).
